### PR TITLE
Add Bubbles Library

### PR DESCRIPTION
**Pull request by ocapraro**
This is a library consisting of simple speech and thought bubbles.
bubbles

### CODE REVIEW

**Comment by dwelle** [comment]
Looks great. Maybe make the farther bubble circle smaller, as I've done in this tweet?

https://twitter.com/excalidraw/status/1367092607076216833

image

**Comment by ocapraro** [comment]
Sure! How does this look?
bubbles

**Review by nrgapple — APPROVED** [review]

File: libraries.json [hunk 1]
```diff
@@ -362,5 +362,18 @@
...
+    "description":  "A collection of simple speech and thought bubbles.",
+    "authors": [
+      {
+        "name": "Oscar Capraro",
+        "url": "https://www.codegrepper.com/app/profile.php?id=44392"
+      }
+    ],
+    "source": "ocapraro/bubbles.excalidrawlib",
+    "preview": "ocapraro/bubbles.png",
+    "date": "2021-03-02"
```
Comment (dwelle) [inline libraries.json:377]:
```suggestion
    "date": "2021-03-03"
```

**Comment by dwelle** [comment]
Looking good. Let's merging it. Thanks!

**Pull request merged**